tests/e2e/features/dicomViewer.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Preview dicom image
+
+  Scenario: user previews dicom image
+    Given "Admin" has logged in
+    And user has uploaded the dicom file "MRBRAIN.dcm"
+    When user previews the dicom file "MRBRAIN.dcm"
+    And user checks VIP metadata for dicom file
+    And user checks the extended metadata for dicom file
+    And user opens files app
+    And user deletes all dicom files
+    And user logs out
```

Comment:
@PrajwolAmatya what do you mean by the step `And the user opens files app`? I guess the files app opens automatically when the user closes the preview. Therefore it might be better to list the step that the user actively does, which would be  `And the user closes the dicom file preview` ?